### PR TITLE
Raise warning not an error on unsupported files

### PR DESCRIPTION
**Pull request by antbertrand**
Added a little condition so that it ignores not supported files, warns the user about it, but doesn't raise any error.
On mac we have those .DS_Store files in every folder that made it difficult to upload local images using deepocli.

### CODE REVIEW

File: deepomatic/cli/json_schema.py [hunk 1]
```diff
@@ -228,3 +228,9 @@ def validate_json(json_data):
                 schema_type = schema_name
                 break
     return is_valid, error, schema_type
+
+
+if __name__ == "__main__":
```
Comment (Alexis-Jacob) [inline deepomatic/cli/json_schema.py:233]: to remove I guess ?

**Review by Alexis-Jacob — APPROVED** [review]
please try to add a small test to cover this new condition

File: deepomatic/cli/json_schema.py (hunk 1 above)
Comment (antbertrand) [reply deepomatic/cli/json_schema.py:233]: oh, yes my bad

File: deepomatic/cli/json_schema.py [hunk 2]
```diff
@@ -228,3 +228,5 @@ def validate_json(json_data):
                 schema_type = schema_name
                 break
     return is_valid, error, schema_type
+
```
Comment (maingoh) [inline deepomatic/cli/json_schema.py:231]: nit: ideally don't add unecessary lines, especially if you don't touch the rest of the file :smile:

**Review by maingoh — APPROVED** [review]
Nice feature !

**Pull request merged**